projects/core/src/lib/components/home-page/home-page.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+	ChangeDetectionStrategy,
+	ChangeDetectorRef,
+	Component,
+	OnDestroy,
+	OnInit
+} from "@angular/core";
+import { Subject, takeUntil } from "rxjs";
+import { HomePageSectionsService } from "../../services/home-page-sections.service";
+import { HomePageSection } from "../../models/home-page-section";
+
+@Component({
+	selector: "jsmu-home-page",
+	templateUrl: "./home-page.component.html",
+	styleUrls: ["./home-page.component.scss"],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class HomePageComponent implements OnInit, OnDestroy {
+	public sections: HomePageSection[] = [];
+	private destroy$: Subject<void> = new Subject<void>();
+
+	constructor(private homePageService: HomePageSectionsService, private ref: ChangeDetectorRef) {}
+
+	public getSections(): void {
+		this.homePageService
+			.getSections<HomePageSection[]>()
```

Comment:
just import sections from assets as variable and use, this does not require http request

service is not needed yet, do not complicate

Reply:
ok,resolved